### PR TITLE
Fixed tests.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

**Comment by terabytesoftw** [comment]
Of course the `composer.json` we will adjust it once merge https://github.com/yiisoft/db/pull/240

**Review by samdark — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -21,17 +21,17 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.4|^8.0",
-        "yiisoft/db": "^3.0@dev",
```
Comment (samdark) [inline composer.json:24]: Shouldn't we still specify `@dev?`

Comment (terabytesoftw) [reply composer.json:24]: It's an aliases it works both ways without problems, but I fix it now.

Comment (terabytesoftw) [reply composer.json:24]: done.

**Pull request merged**